tests/components/prowl/test_yaml_deprecation.py

```diff
@@ -0,0 +1,177 @@
+"""Tests for the migration from YAML to config entries."""
+
+from unittest.mock import AsyncMock
+
+import prowlpy
+import pytest
+
+from homeassistant.components import notify
+from homeassistant.components.prowl.const import DOMAIN
+from homeassistant.const import CONF_API_KEY
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import issue_registry as ir
+from homeassistant.setup import async_setup_component
+
+from .conftest import OTHER_API_KEY, TEST_API_KEY, TEST_NAME, TEST_SERVICE
+from .helpers import get_config_entry
+
+from tests.common import MockConfigEntry
+
+SERVICE_DATA = {"message": "Test Notification", "title": "Test Title"}
+
+
+@pytest.mark.usefixtures("configure_prowl_through_yaml")
+async def test_yaml_migration_creates_config_entry(
+    hass: HomeAssistant, issue_registry: ir.IssueRegistry
+) -> None:
+    """Test that YAML configuration triggers config entry creation."""
+    entry = get_config_entry(hass, TEST_API_KEY, config_method="import")
```

Comment:
Instead of making a lookup, we can also just let the `configure_prowl_through_yaml` fixture return the config entry.

../Frenck

Reply:
Is there a method of getting the ConfigEntry after running `async_setup_component` that is different from doing essentially `for entry in hass.config_entries.async_entries(DOMAIN):`? (or maybe you just suggested to move it to the fixture to avoid the helper function?)